deployments/properties/terraform.properties.json

```diff
@@ -3,5 +3,5 @@
     "description": "Set up Terraform CLI in your GitHub Actions workflow.",
     "creator": "HashiCorp",
     "iconName": "terraform",
-    "categories": ["Deployment"]
+    "categories": ["Deployment", "Dockerfile"]
```

Comment:
Terraform template does not look like related to dockerfile. Or am I missing something?

Reply:
Done.